literalai/event_processor.py

```diff
@@ -98,6 +98,7 @@ def _process_batch(self, batch: List):
             self.processing_counter -= len(batch)
 
     def flush_and_stop(self):
+        time.sleep(4)
```

Comment:
Found an issue where the llm step is added to the queue but not flushed, would like to solve this.